### ISSUE TITLE
Incompatibility with memoir

### ISSUE THREAD

**Issue by rolfn** [rolfn/xltabular#10]
Lasse Rosing reported the following bug:
```
\listfiles
\documentclass[a4paper]{memoir}
\usepackage{booktabs,xltabular}

\begin{document}

\begin{tabularx}{\textwidth}{@{}XXXX@{}}
KPI & Index & Other & Other \\ \toprule
foo & bar & baz & foobar \\ \hline
foo & bar & baz & foobar \\
foo & bar & baz & foobar \\ \bottomrule
\end{tabularx}

\end{document}
```
The lines are showing only in the first column. Seems to be an incompatibility with memoir.

**Comment by rolfn** [comment]
Adding 
```
\expandafter\let\csname ver@booktabs.sty\endcsname\relax
```
just before `\usepackage{booktabs,xltabular}` show the effect of  "memoir".

**Comment by rolfn** [comment]
This problem is already fixed in "memoir" v3.7k, 2020/02/06.
